Doc/tools/extensions/pyspecific.py

```diff
@@ -288,6 +288,9 @@ def run(self):
         version_deprecated = expand_version_arg(self.arguments[0],
                                                 self.config.release)
         version_removed = self.arguments.pop(1)
+        if version_removed == 'next':
+            raise ValueError(
+                'deprecated-removed:: second argument cannot be `next`')
```

Comment:
```suggestion
        if version_removed == 'next':
            raise ValueError(
                'deprecated-removed:: second argument should be the version in which '
                'the API will be removed, and cannot be `next`')
```

Reply:
Thanks for the suggestion! But I do think it's too long, and I hope anyone who adds `deprecated-removed::` already knows what the args are. If they need this error to find that out, something's very wrong :)